client/src/components/App/Content/InvestigationForm/TabManagement/ContactQuestioning/ContactQuestioningClinical.tsx

```diff
@@ -111,50 +114,52 @@ const ContactQuestioningClinical: React.FC<Props> = (props: Props): JSX.Element
                     <Grid container>
-                        <FieldName xs={5} fieldName='קרבה משפחתית:' />
-                        <Grid item xs={7}>
+                        <FieldName xs={5} fieldName={ContactQuestioningFieldsNames.FAMILY_RELATIONSHIP} className={classes.fieldName}/>
+                        <Grid item xs={5}>
                             <Controller
                                 control={control}
                                 name={`form[${index}].${InteractedContactFields.FAMILY_RELATIONSHIP}`}
-                                defaultValue={interactedContact.familyRelationship}
+                                defaultValue={interactedContact.familyRelationship}                                            
                                 render={(props) => {
-                                    return (<FormControl>
-                                        <Select
-                                            {...props}
-                                            disabled={isFieldDisabled || isFamilyContact}
-                                            test-id='familyRelationshipSelect'
-                                            placeholder='קרבה משפחתית'
-                                            onChange={(event) => {
-                                                props.onChange(event.target.value)
-                                            }}
-                                        >
-                                            {
-                                                familyRelationships?.length > 0 &&
-                                                [emptyFamilyRelationship].concat(familyRelationships).map((familyRelationship) => (
-                                                    <MenuItem className={classes.menuItem}
-                                                        key={familyRelationship.id}
-                                                        value={familyRelationship.id}>
-                                                        {familyRelationship.displayName}
-                                                    </MenuItem>
-                                                ))
-                                            }
-                                        </Select>
-                                    </FormControl>)
+                                    return (
+                                        <FormControl variant='outlined' fullWidth>
+                                            <Select
+                                                {...props}
+                                                disabled={isFieldDisabled || isFamilyContact}
+                                                test-id='familyRelationshipSelect'
+                                                placeholder='קרבה משפחתית'
+                                                onChange={(event) => {
+                                                    props.onChange(event.target.value)
```

Comment:
Same here, maybe it's more worthwhile to try and break it to see what can happen

Reply:
I tried, it seems to work good :)